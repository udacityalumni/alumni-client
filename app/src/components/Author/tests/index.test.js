import Author from '../index';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import React from 'react';

describe('<Author />', () => {
  it('should render with default props', () => {
    const wrapper = shallow(
      <Author
        author={{ name: 'Ryan Collins', avatar: 'ryancollins.io/avarat', bio: 'Hello world' }}
      />
    );
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
