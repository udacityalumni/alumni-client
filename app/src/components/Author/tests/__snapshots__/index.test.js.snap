exports[`<Author /> should render with default props 1`] = `
<Article
  align="center"
  direction="column"
  pad="none">
  <Heading
    align="center"
    tag="h2">
    About the Author
  </Heading>
  <Box
    align="center"
    announce={false}
    direction="column"
    focusable={true}
    pad="none"
    responsive={true}
    tag="div">
    <img
      src="ryancollins.io/avarat" />
  </Box>
  <Heading
    align="center"
    tag="h3">
    Ryan Collins
  </Heading>
  <Quote
    borderColorIndex="light-2"
    emphasizeCredit={true}
    pad={
      Object {
        "horizontal": "large",
        "vertical": "small"
      }
    }
    size="large">
    <Heading
      align="center"
      tag="h5">
      Hello world
    </Heading>
  </Quote>
</Article>
`;
